server.js

```diff
@@ -39,16 +41,22 @@ app.prepare().then(() => {
       res.writeHead(301, { Location: 'https://dvc.org' + doc_pathname })
       res.end()
     } else if (/^(code|data|remote)\.dvc\.org$/.test(req.headers.host)) {
+      // {code/data/remote}.dvc.org -> corresponding S3 bucket
       res.writeHead(301, {
         Location:
           'https://s3-us-east-2.amazonaws.com/dvc-public/' +
           req.headers.host.split('.')[0] +
           pathname
       })
       res.end()
-
-      // Special URL path redirects below
+    } else if (/^\/doc\/commands-reference.*/.test(pathname)) {
```

Comment:
it will match `/commands-referencesdsdsdsd/something`, let's make it something like

`/^\/doc\/commands-reference(\/.*/)?`

?

Reply:
Sure, just need to make that `/` optional for path `/doc/commands-reference`. Fixed in 1eaca99.

Reply:
Oh wait I see your regex also achieves that. Looks better let me try that...

Reply:
Yes. (Except your last `/` should be after `?` 😋  Updated and tried with paths `/doc/commands-reference`, `/doc/commands-reference/add`, and `/doc/commands-referenceadd`. see latest commit.